community/modules/scheduler/schedmd-slurm-gcp-v6-controller/partition.tf

```diff
@@ -66,23 +66,22 @@ module "slurm_nodeset_template" {
   tags                     = concat([local.slurm_cluster_name], each.value.tags)
 }
 
-module "slurm_nodeset" {
-  source   = "github.com/GoogleCloudPlatform/slurm-gcp.git//terraform/slurm_cluster/modules/slurm_nodeset?ref=6.5.13"
-  for_each = local.nodeset_map
-
-  instance_template_self_link = module.slurm_nodeset_template[each.key].self_link
-
-  enable_placement       = each.value.enable_placement
-  maintenance_interval   = each.value.maintenance_interval
-  network_tier           = each.value.network_tier
-  node_count_dynamic_max = each.value.node_count_dynamic_max
-  node_count_static      = each.value.node_count_static
-  nodeset_name           = each.value.nodeset_name
-  node_conf              = each.value.node_conf
-  subnetwork_self_link   = each.value.subnetwork_self_link
```

Comment:
Are we moving from subnetwork_self_link to subnetwork  ?

Reply:
No, I'm just matching what script expects from stored config.